### PR TITLE
Feature/gdev86

### PR DESCRIPTION
**Pull request by KerstenBreuer**
Add template for README.

Please note, (as #3) this PR depends on the shared lib at https://github.com/ghga-de/ghga-service-chassis-lib/tree/feature/rest which is not yet available at PyPI. Thus to test it now run the following:

```
pip install git+https://github.com/ghga-de/ghga-service-chassis-lib@feature/rest
dev_install
```

This is also why the CI currently fails. Once the shared library is realized, the installation will work as intended.

This PR depends on #1 and #3, please review them first.

### CODE REVIEW

**Review by ac-jorellanaf — APPROVED** [review]

File: .github/workflows/dev_cd.yml [hunk 1]
```diff
@@ -54,23 +54,17 @@ jobs:
   #   steps:
   #     - uses: actions/checkout@v2
   #       name: Check out code
-  #     - uses: docker/setup-qemu-action@v1
-  #       name: Set up QEMU
-  #     - uses: docker/setup-buildx-action@v1
-  #       name: Set up Docker Buildx
-  #     - uses: docker/login-action@v1
-  #       name: Login to DockerHub
+  #     - uses: mr-smithers-excellent/docker-build-push@v5
```
Comment (ac-jorellanaf) [inline .github/workflows/dev_cd.yml:57]: Dumb question, but why did we change the library here?

**Review by ac-jorellanaf — APPROVED** [review]

File: .github/workflows/dev_cd.yml (hunk 1 above)
Comment (KerstenBreuer) [reply .github/workflows/dev_cd.yml:57]: There is apparently an official version for this directly from Github. The one I was using before was fine but from a private person. Maybe we trust Github more, but really no special reason.

File: README.md [hunk 2]
```diff
@@ -20,3 +20,112 @@ This is just a template and needs some adaption to your specific use case.
...
+[corresponding section](https://pydantic-docs.helpmanual.io/usage/settings/#secret-support)
+of the pydantic documentation.
+
+
+## Development
+For setting up the development environment, we rely on the
+[devcontainer feature](https://code.visualstudio.com/docs/remote/containers) of vscode
+in combination with Docker Compose.
+
+To use it, you have to have Docker Compose as well as vscode with its "Remote - Containers" extension (`ms-vscode-remote.remote-containers`) extension installed.
```
Comment (lkuchenb) [inline README.md:96]:
```suggestion
To use it, you have to have Docker Compose as well as vscode with its "Remote - Containers" extension (`ms-vscode-remote.remote-containers`) installed.
```

File: README.md [hunk 3]
```diff
@@ -20,3 +20,112 @@ This is just a template and needs some adaption to your specific use case.
...
+Then, you just have to open this repo in vscode and run the command
```
Comment (lkuchenb) [inline README.md:97]:
```suggestion
Then open this repository in vscode and run the command
```

File: README.md [hunk 4]
```diff
@@ -20,3 +20,112 @@ This is just a template and needs some adaption to your specific use case.
...
+- infrastructural dependencies of the service (databases, etc.)
+- all relevant vscode extensions pre-installed
+- pre-configured linting and auto-formating
+- a pre-configured debugger
+- automatic license-header insertion
+
+Moreover, inside the devcontainer, there are two convenience commands available
+(please type them in the integrated terminal of vscode):
+- `dev_install` - install the service with all development dependencies,
+installs pre-commit, and applies any migration scripts to the test database
```
Comment (lkuchenb) [inline README.md:110]: BTW, why is pre-commit not installed in the Dockerfile?

File: README.md [hunk 5]
```diff
@@ -20,3 +20,112 @@ This is just a template and needs some adaption to your specific use case.
...
+(please run that if you are starting the devcontainer for the first time
+or if added any python dependencies to the [`./setup.cfg`](./setup.cfg))
```
Comment (lkuchenb) [inline README.md:112]:
```suggestion
or if you added any python dependencies to the [`./setup.cfg`](./setup.cfg))
```

**Review by lkuchenb — APPROVED** [review]
👍  Minor remarks below

File: README.md (hunk 4 above)
Comment (KerstenBreuer) [reply README.md:110]: The workspace is mounted after the container builds. And precommit installs as a git hook to the .git directory.

Comment (lkuchenb) [reply README.md:110]: 👍

**Pull request merged**